ingress/proxy.go

```diff
@@ -0,0 +1,16 @@
+package ingress
+
+import "github.com/sky-uk/feed/controller"
+
+// Proxy proxies traffic from external clients to the internal services in Kubernetes.
+type Proxy interface {
```

Comment:
Why do we need another interface for proxy? Is this just not an implementation of the Upater interface?